plans/migrate.pp

```diff
@@ -0,0 +1,31 @@
+plan peadm::migrate (
+  Peadm::SingleTargetSpec $old_primary_host,
+  Peadm::SingleTargetSpec $new_primary_host,
+) {
+  peadm::assert_supported_bolt_version()
+
+  $backup_file = run_plan('peadm::backup', $old_primary_host, {
+      backup_type => 'migration',
+  })
+
+  download_file($backup_file['path'], 'backup', $old_primary_host)
+
+  $backup_filename = basename($backup_file['path'])
+  $remote_backup_path = "/tmp/${backup_filename}"
+  $current_dir = system::env('PWD')
+
+  upload_file("${current_dir}/downloads/backup/${old_primary_host}/${backup_filename}", $remote_backup_path, $new_primary_host)
+
+  run_plan('peadm::install', {
+      primary_host                => $new_primary_host,
+      console_password            => 'Puppetlabs123!',
```

Comment:
we might need a ticket to grab this form the old_primary_host if we can but will do for now